docs/src/pages/components/Button/AndroidButton.mdx

```diff
@@ -9,78 +9,39 @@ githubPath: button
 import AndroidComponentScreenshots from 'components/Screenshots/AndroidComponentScreenshots';
 
 import ButtonReadme from 'backpack-android/docs/view/Button/README.md';
-import LinkReadme from 'backpack-android/docs/view/ButtonLink/README.md';
 
-import screenshotPrimary from 'backpack-android/docs/view/Button/screenshots/primary.png';
-import screenshotPrimaryDm from 'backpack-android/docs/view/Button/screenshots/primary_dm.png';
-import screenshotPrimaryOnDark from 'backpack-android/docs/view/Button/screenshots/primary_on_dark.png';
-import screenshotPrimaryOnDarkDm from 'backpack-android/docs/view/Button/screenshots/primary_on_dark_dm.png';
-import screenshotPrimaryOnLight from 'backpack-android/docs/view/Button/screenshots/primary_on_light.png';
-import screenshotPrimaryOnLightDm from 'backpack-android/docs/view/Button/screenshots/primary_on_light_dm.png';
-import screenshotSecondary from 'backpack-android/docs/view/Button/screenshots/secondary.png';
-import screenshotSecondaryDm from 'backpack-android/docs/view/Button/screenshots/secondary_dm.png';
-import screenshotDestructive from 'backpack-android/docs/view/Button/screenshots/destructive.png';
-import screenshotDestructiveDm from 'backpack-android/docs/view/Button/screenshots/destructive_dm.png';
-import screenshotFeatured from 'backpack-android/docs/view/Button/screenshots/featured.png';
-import screenshotFeaturedDm from 'backpack-android/docs/view/Button/screenshots/featured_dm.png';
-import screenshotLink from 'backpack-android/docs/view/ButtonLink/screenshots/default.png';
-import screenshotLinkDm from 'backpack-android/docs/view/ButtonLink/screenshots/default_dm.png';
+import Default from 'backpack-android/docs/view/Button/screenshots/standard.png';
+import DefaultDm from 'backpack-android/docs/view/Button/screenshots/standard_dm.png';
+import Large from 'backpack-android/docs/view/Button/screenshots/large.png';
+import LargeDm from 'backpack-android/docs/view/Button/screenshots/large_dm.png';
+import Link from 'backpack-android/docs/view/Button/screenshots/link.png';
+import LinkDm from 'backpack-android/docs/view/Button/screenshots/link_dm.png';
 
 ## Table of contents
 
 ## Primary
```

Comment:
Is this still needed? I think the table of contents is slightly misaligned now? 😄

Reply:
good catch!